app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,28 @@
+class CommentsController < ApplicationController
+  before_action :find_post
+
+  def create
+    # @user = User.find(params[:author_id])
```

Comment:
- [ ] I would recommend removing the commented code to make the codebase cleaner.

Reply:
Thanks, @Dipeshtwis for your kind suggestion and feedback!. This suggestion is implemented on commit 54ec112.